analytics-listeners/src/main/java/org/exoplatform/analytics/listener/wiki/WikiPageListener.java

```diff
@@ -103,7 +103,7 @@ private void createWikiPageStatistic(Page page,
                                        String operation,
                                        PageUpdateType wikiUpdateType) {
     StatisticData statisticData = new StatisticData();
-    statisticData.setModule("Note");
+    statisticData.setModule("Notes");
```

Comment:
you shouldn't change the collected data points names, almost never :-)
Here we will need to use I18N for those labels. (It's pure UI, no services change)